core/tests/org.fusesource.ide.camel.tests.util/src/org/fusesource/ide/camel/test/util/editor/AbstractCamelEditorIT.java

```diff
@@ -70,7 +70,7 @@ public AbstractCamelEditorIT() {
 	
 	@BeforeClass
 	public static void beforeClass() throws Throwable {
-		fuseProject = new FuseProject(AbstractCamelEditorIT.class.getName());
+		fuseProject = new FuseProject();
```

Comment:
this one cannot use the no-arg constructor because it is not used as a JUnit Rule.